crates/e2e/src/prover/block.rs

```diff
@@ -150,30 +149,25 @@ pub async fn parallel_prover(
     // collect the trusted height and root to then supply them optimistically to the prover
     for block_number in start_height..(start_height + num_blocks) {
         println!("\nProcessing block: {block_number}");
-        
```

Comment:
looks like this is removing all the spacing we added in the previous PR? maybe because it was a github suggested comment it was applied with additional whitespace accidentally and then removed by the formatter.